protocol-configuration/mainnetAddresses.ts

```diff
@@ -66,39 +56,6 @@ export const MainnetContractsConfig = {
     address: '0x6F6580285a63f1e886548458f427f8695BA1a563',
     category: AddressCategory.Core
   },
-  guardianMultisig: {
-    artifactName: 'unknown',
-    address: '0xB8f482539F2d3Ae2C9ea6076894df36D1f632775',
-    category: AddressCategory.Security
-  },
-  feiDAO: {
-    artifactName: 'FeiDAO',
-    address: '0x0BEF27FEB58e857046d630B2c03dFb7bae567494',
-    category: AddressCategory.Governance
-  },
-
-  feiDAOTimelock: {
-    artifactName: 'FeiDAOTimelock',
-    address: '0xd51dbA7a94e1adEa403553A8235C302cEbF41a3c',
-    category: AddressCategory.Governance
-  },
-  rariTimelock: {
-    artifactName: 'Timelock',
-    address: '0x8ace03Fc45139fDDba944c6A4082b604041d19FC',
-    category: AddressCategory.Governance
-  },
-
-  tribeRariDAO: {
-    artifactName: 'FeiDAO',
-    address: '0x637deEED4e4deb1D222650bD4B64192abf002c00',
-    category: AddressCategory.Governance
-  },
-
-  rariGovernanceProxyAdmin: {
-    artifactName: 'ProxyAdmin',
-    address: '0x1c9aA54a013962C2444ECae06902F31D532c6AD3',
-    category: AddressCategory.Governance
-  },
 
   rariTimelockFeiOldLens: {
```

Comment:
this should be deprecated bc the entire CR oracle is considered deprecated and this contract has no more changes

Reply:
Got it, updated